build.js

```diff
@@ -64,16 +64,78 @@ function getOutputPath(pkg) {
 	return `./web/js/dist/${pkgName}.min.js`;
 }
 
+/** @type {lightningcss.Visitor} */
+const sizeToWidthHeight = {
```

Comment:
This is cool, not sure worth the extra complexity though? We could remove use of these non-standard values instead? Your call

Reply:
I kept it because there was a postcss plugin for it, so I assumed it must've been useful. However, I agree we should probably remove them. It's only used in like 4 places anyways. I'll remove it later tonight!